src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (x) => {
+    return (y) => {
+      counter++;
+
+      return (counter > 3 && counter % 2 === 0)
```

Comment:
A variable can be declared here for even calls just to improve readability. 
```suggestion
     const evenCalls = counter > 3 && counter % 2 === 0
      return evenCalls
```

Reply:
@nurdanokcu i like the direction, but you should consider the naming of the booleans. `isCallEven` is better, but I would make it `shouldBreak` her